docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+
+![The ETH ingress rate panel](/images/docs/grafana/eth-ingress-rate.png)
+
+#### ETH egress data rate
+
+Egress is the process of data leaving the local node and being transferred to its peers. This panel shows the rate that data specifically using the eth subprotocol is leaving the local node in units of kB/s (kilobytes per second). Make sure your dashboard includes the latest version of the eth subprotocol!
+
+![The ETH egress rate panel](/images/docs/grafana/eth-egress-rate.png)
+
+#### ETH ingress traffic
+
+Ingress is the process of data arriving at the local node from its peers. This panel shows a moment-by-moment snapshot of the amount of data that is arriving at the local node, specifically using the eth subprotocol, in units of GB (gigabytes). Make sure your dashboard includes the latest version of the eth subprotocol!
+
+![The ETH ingress traffic panel](/images/docs/grafana/eth-ingress-traffic.png)
+
+#### ETH egress traffic
+
+Egress is the process of data leaving the local node and being transferred to its peers. This panel shows a moment-by-moment snapshot of the amount of data that is leaving the local node, specifically using the eth subprotocol, in units of GB (gigabytes). Make sure your dashboard includes the latest version of the eth subprotocol!
+
+![The ETH egress traffic panel](/images/docs/grafana/eth-egress-traffic.png)
+
+### Blockchain
+
+Panels in the Blockchain category track the local node's view of the blockchain.
+
+#### Chain head
+
+The chain head simply tracks the latest block number that the local node is aware of.
+
+![The Chain head panel](/images/docs/grafana/chain-head.png)
+
+#### Transaction pool
+
+Geth has a capacity for pending transactions defined by `--txpool.globalslots` (default is 5160). The number of slots filled with transactions is tracked as `slots`. The transactions in the pool are divided into pending transactions and queued transactions. Pending transactions are ready to be processed and included in a block, whereas queued transactions are those whose transaction nonces are out of sequence. Queued transactions can become pending transactions if transactions with the missing nonces become available. In the dashboard pending transactions are labelled as `executable` and queued transactions are labelled `gapped`. The subset of those global transactions that originated from the local node are tracked as `local`.
+
+![The tx pool panel](/images/docs/grafana/tx-pool.png)
+
+#### Block processing
+
+The block processing panel tracks the time taken to complete the various tasks involved in processing each block, measured in microseconds or nanoseconds. Specifically, this includes:
+
+- **execution**: time taken to execute the transactions in the block
+- **validation**: time taken to compute a new state root and compare it to the one that arrived in the block
```

Comment:
validation doesn't include the time spent on the state root computation(storage trie hashing + account trie hashing)

Reply:
Is this accurate?
```suggestion
- **validation**: time taken to validate that the information in a received block body matches what is described in the block header. 
```